common/dal-context/src/lib.rs

```diff
@@ -0,0 +1,153 @@
+// Copyright 2022 Datafuse Labs.
```

Comment:
It would be better we split lib.rs into some sprerate files, to make the lib.rs clean.
If we put them in lib.rs file, guessing that more and more codes will be put in the future.

Reply:
Makes sense, I will split them.

Reply:
Addressed, PTAL.